### PR TITLE
Fix the validation of aggregate and proof

### LINKED ISSUE #691: AggregateAndProof gossip validation isn't working
To reproduce:
- uncomment validation
- run and connect two beacon nodes as per readme

My best guess is that we should fetch state/advance state matching attestation slot or something

### PR DESCRIPTION
**Pull request by twoeths**
## Goal
+ Close #691

### CODE REVIEW

File: packages/lodestar/src/network/gossip/validator.ts [hunk 1]
```diff
@@ -85,48 +90,46 @@ export class GossipMessageValidator implements IGossipMessageValidator {
...
+    const attestorIndices = getAttestingIndices(this.config, state,
+      aggregationAndProof.aggregate.data, aggregationAndProof.aggregate.aggregationBits);
+    if (!attestorIndices.includes(aggregationAndProof.aggregatorIndex)) {
+      return false;
+    }
+    if (!isAggregator(
+      this.config,
+      state,
+      slot,
+      aggregationAndProof.aggregate.data.index,
```
Comment (twoeths) [inline packages/lodestar/src/network/gossip/validator.ts:117]: This is main fix, should be committee index instead of aggregator index

**Review by mpetrunic — CHANGES_REQUESTED** [review]
Can you write test cases for this, I think it also shouldn't use latest state but not 100%

**Comment by twoeths** [comment]
I think we'll end up with using the same state like the one to validate block in my other PR #710 . Need to apply for other validation too as long as we have a `slot`.

**Comment by twoeths** [comment]
Replaced by #760

**Pull request closed without merging**